src/Estimators/InputSection.h

```diff
@@ -173,8 +197,8 @@ class InputSection
 private:
   // Query functions
   bool isAttribute(const std::string& name) const { return attributes.find(name) != attributes.end(); }
-  bool isParameter(const std::string& name) const { return parameters.find(name) != parameters.end(); }
   bool isDelegate(const std::string& name) const { return delegates.find(name) != delegates.end(); }
```

Comment:
I don't know what is Delegate. So upcoming documentation needs to explain this and give examples.

Reply:
This is for sure.  Without example its hard to explain.  But bringing the concrete example in results in too much.  This could be another instance where the name is too general and we may want to return to the naming after the docs are expanded.